packages/react/src/QuestionnaireBuilder/QuestionnaireBuilder.tsx

```diff
@@ -8,15 +8,21 @@ import {
   QuestionnaireItemAnswerOption,
   Reference,
 } from '@medplum/fhirtypes';
+import { IconArrowDown, IconArrowUp } from '@tabler/icons-react';
 import React, { useEffect, useRef, useState } from 'react';
 import { Form } from '../Form/Form';
 import { useMedplum } from '../MedplumProvider/MedplumProvider';
+import { QuestionnaireFormItem } from '../QuestionnaireForm/QuestionnaireFormItem/QuestionnaireFormItem';
 import { getValueAndType } from '../ResourcePropertyDisplay/ResourcePropertyDisplay';
 import { ResourcePropertyInput } from '../ResourcePropertyInput/ResourcePropertyInput';
 import { useResource } from '../useResource/useResource';
 import { killEvent } from '../utils/dom';
 import { isChoiceQuestion, QuestionnaireItemType } from '../utils/questionnaire';
-import { QuestionnaireFormItem } from '../QuestionnaireForm/QuestionnaireFormItem/QuestionnaireFormItem';
+
+enum Direction {
+  UP,
```

Comment:
I'm not sure if this is useful here, but I've found that using `delta` with values `1` or `-1` can be more elegant than direction enums.  Then it becomes less `if`/`else`, and just math.